src/components/ae-fraction/ae-fraction.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <span class="ae-fraction" :class="[ fill ]">
+    <sup>{{ numerator }}</sup>
+    &frasl;
+    <sub>{{ denominator }}</sub>
+  </span>
+</template>
+<script>
+export default {
+  name: 'ae-fraction',
+  props: {
+    /**
+     * Fill property changes the color state of the fraction component.
+     * Select something between: `primary, secondary, neutral, alternative, white, black`
+     */
+    fill: {
+      type: String,
+      validator: value => [
+        'primary',
+        'secondary',
+        'neutral',
+        'alternative',
+        'white',
+        'black',
+      ].includes(value),
+    },
+
+    /**
+     * The numerator property defines
+     * the upper number in the fraction.
+     */
+    numerator: {
+      type: [Number, String],
+      default: 0,
+    },
+
+    /**
+     * The numerator property defines
+     * the lower number in the fraction.
+     */
+    denominator: {
+      type: [Number, String],
+      default: 0,
+    },
+  },
+};
+</script>
+<style lang="scss" scoped>
+@import '../../styles/variables/colors';
+@import '../../styles/variables/typography';
+
+.ae-fraction {
+  display: inline-block;
+  font-family: $font-sans;
+  font-weight: 500;
+  font-style: normal;
+  font-stretch: normal;
+  line-height: 0;
```

Comment:
Why `line-height` is `0`?

Reply:
Just to enforce line-hight to 0

Reply:
is it necessary? can we remove it?

Reply:
Yes, there are high chances that the element line-hight can change in different context.

Reply:
Does line-height can break <sup> and <sub>? If not, we don't have to worry about line-height.

Reply:
removed.